builtin/providers/google/resource_compute_forwarding_rule.go

```diff
@@ -93,13 +127,24 @@ func resourceComputeForwardingRuleCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	ps := d.Get("ports").(*schema.Set).List()
+	ports := make([]string, 0, len(ps))
+	for _, v := range ps {
+		ports = append(ports, v.(string))
+	}
+
 	frule := &compute.ForwardingRule{
-		IPAddress:   d.Get("ip_address").(string),
-		IPProtocol:  d.Get("ip_protocol").(string),
-		Description: d.Get("description").(string),
-		Name:        d.Get("name").(string),
-		PortRange:   d.Get("port_range").(string),
-		Target:      d.Get("target").(string),
+		BackendService:      d.Get("backend_service").(string),
```

Comment:
You're totally right, I misunderstood the situations in which `d.Get` would return nil. Whoops! Thanks for correcting me. :D